etcdctl/command/mk_command.go

```diff
@@ -51,7 +51,9 @@ func mkCommandFunc(c *cli.Context, ki client.KeysAPI) {
 	ttl := c.Int("ttl")
 
 	ctx, cancel := contextWithTotalTimeout(c)
-	resp, err := ki.Set(ctx, key, value, &client.SetOptions{TTL: time.Duration(ttl) * time.Second, PrevExist: client.PrevIgnore})
+	// Since PrevNoExist means that the Node must not exist,
+	// this Set method would return error if the key already exists.
```

Comment:
`this Set method always creates a new key.` Write about its benefits, which matches what `mk` command does.